src/app/rest/support.service.ts

```diff
@@ -25,7 +25,7 @@ export class SupportRestService {
     }
 
     private getRestServiceRoot():string {
-        return REST_SERVICE_ROOT + "/support";
+        return REST_SERVICE_V1_ROOT + "/support";
```

Comment:
Just update support service path with v2 and don't replace to many places in here